packages/blocks/src/page-block/default/utils.ts

```diff
@@ -539,6 +539,24 @@ export function createDragHandle(pageBlock: DefaultPageBlockComponent) {
       pageBlock.selection.state.type = dragging ? 'block:drag' : 'block';
     },
     setSelectedBlock(modelState: EditingState | null) {
+      if (modelState) {
```

Comment:
Instead of handle database selection here, maybe we can move this into database service.